scripts/bash/gitlab_migrate.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
Can you break the logic into functions?

Reply:
Same answer as for the [gitlab_install.sh] file